inc/abstracttarget.class.php

```diff
@@ -696,6 +696,17 @@ protected function prepareActors(PluginFormcreatorForm $form, PluginFormcreatorF
 
                $userIds = [$object->fields[$groupFk]];
                break;
+
+            case PluginFormcreatorTarget_Actor::ACTOR_TYPE_AUTHORS_SUPERVISOR:
+               $requester_id = $formanswer->fields['requester_id'];
+
+               $user = new User;
+               $users = $user->find(['id' => $requester_id]);
+               if (!empty($users)) {
+                  $userIds = [$users[$requester_id]['users_id_supervisor']];
+               }
+               // $userIds = [$formanswer->fields['requester_id']];
```

Comment:
$user = User::getById($requester_id);
```suggestion
               $user = User::getById($requester_id);
               if ($user !== false) {
                  $userIds = [$user->fields['users_id_supervisor']];
               }
```

Reply:
Thanks, it works.